### PR TITLE
Fix for Jira issues : EMM-1212,EMM-1214

### PR DESCRIPTION
**Pull request by Kamidu**

### CODE REVIEW

File: modules/apps/jaggery/emm/src/emm/units/device-detail/device-detail.hbs [hunk 1]
```diff
@@ -151,7 +151,7 @@
                                         </div>
                                         <div id="policy-list-container">
                                             <div class="panel-body">
-                                                Not available yet
+                                                No active policies
```
Comment (harshanL) [inline modules/apps/jaggery/emm/src/emm/units/device-detail/device-detail.hbs]: What about something like "There are no active policies for this device" ?

File: modules/apps/jaggery/emm/src/emm/units/device-detail/device-detail.hbs [hunk 2]
```diff
@@ -203,7 +203,7 @@
                                         </div>
                                         <div id="operations-log-container">
                                             <div class="panel-body">
-                                                Not available yet
+                                                No oparations performed yet
```
Comment (harshanL) [inline modules/apps/jaggery/emm/src/emm/units/device-detail/device-detail.hbs]: What about something like "There are no operations performed on this device" ?

File: modules/apps/jaggery/emm/src/emm/units/device-detail/device-detail.hbs (hunk 1 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/units/device-detail/device-detail.hbs]: noted

File: modules/apps/jaggery/emm/src/emm/units/device-detail/device-detail.hbs (hunk 2 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/units/device-detail/device-detail.hbs]: noted

**Pull request merged**